src/ui/server/helpers/sanitise-data/index.test.ts

```diff
@@ -99,13 +99,51 @@ describe('server/helpers/sanitise-data', () => {
     });
   });
 
+  describe('isDayMonthYearField', () => {
+    describe('when a field name contains `-day`', () => {
+      it('should return true', () => {
```

Comment:
I've been thinking about this as we do these checks in other places. I don't think there is a need to because of typescript. If this function is attempted to be called from anywhere without `fieldName` string parameter, the code will not compile.

So I don't think there is any value in doing the param check and adding a unit test for this with `@ts-ignore` (for not providing a string param), because outside of tests, typescript will prevent the code from compiling.

What do you think?